src/qt/optionsmodel.h

```diff
@@ -44,29 +44,30 @@ class OptionsModel : public QAbstractListModel
     explicit OptionsModel(QObject *parent = nullptr, bool resetSettings = false);
 
     enum OptionID {
-        StartAtStartup,         // bool
-        ShowTrayIcon,           // bool
-        MinimizeToTray,         // bool
-        MapPortUPnP,            // bool
-        MapPortNatpmp,          // bool
-        MinimizeOnClose,        // bool
-        ProxyUse,               // bool
-        ProxyIP,                // QString
-        ProxyPort,              // int
-        ProxyUseTor,            // bool
-        ProxyIPTor,             // QString
-        ProxyPortTor,           // int
-        DisplayUnit,            // BitcoinUnits::Unit
-        ThirdPartyTxUrls,       // QString
-        Language,               // QString
-        UseEmbeddedMonospacedFont, // bool
-        CoinControlFeatures,    // bool
-        ThreadsScriptVerif,     // int
-        Prune,                  // bool
-        PruneSize,              // int
-        DatabaseCache,          // int
-        SpendZeroConfChange,    // bool
-        Listen,                 // bool
+        StartAtStartup,               // bool
+        ShowTrayIcon,                 // bool
+        MinimizeToTray,               // bool
+        MapPortUPnP,                  // bool
+        MapPortNatpmp,                // bool
+        MinimizeOnClose,              // bool
+        ProxyUse,                     // bool
+        ProxyIP,                      // QString
+        ProxyPort,                    // int
+        ProxyUseTor,                  // bool
+        ProxyIPTor,                   // QString
+        ProxyPortTor,                 // int
+        DisplayUnit,                  // BitcoinUnits::Unit
+        ThirdPartyTxUrls,             // QString
+        Language,                     // QString
+        UseEmbeddedMonospacedFont,    // bool
+        PeersTabAlternatingRowColors, // bool
+        CoinControlFeatures,          // bool
+        ThreadsScriptVerif,           // int
+        Prune,                        // bool
+        PruneSize,                    // int
+        DatabaseCache,                // int
+        SpendZeroConfChange,          // bool
+        Listen,                       // bool
         OptionIDRowCount,
```

Comment:
Nit in 217d42b7465bf210511ef6cdff0475eb0a9e078c

While here, you could document the type here is `int`.

Additionally, if you have to retouch, you could rename the commit from:
`gui: Add "Alternating Row Color" settings for the Peers Tab `

to:

`qt: Add "Alternating Row Color" settings for the Peers Tab `

Reply:
> While here, you could document the type here is `int`.

I don't think it is required to document the type of a counter, as it is not an option id.

Reply:
> Additionally, if you have to retouch, you could rename the commit from:
> `gui: Add "Alternating Row Color" settings for the Peers Tab `
> 
> to:
> 
> `qt: Add "Alternating Row Color" settings for the Peers Tab `

Done in the [recent](https://github.com/bitcoin-core/gui/pull/307#issuecomment-859324593) push.